File: code/load.mnist_enhanced.py
import os, pandas, h2o
h2o.init()

#If next line fails, set path to location of datasets.
path = os.path.dirname(__file__)
train = h2o.import_file( os.path.join(
  path, "../datasets/mnist.enhanced_train.csv.gz") )
valid = h2o.import_file( os.path.join(
  path, "../datasets/mnist.enhanced_valid.csv.gz") )
test = h2o.import_file( os.path.join(
  path, "../datasets/mnist.enhanced_test.csv.gz") )

x = range(0,896)
y = 897

train[[y]] = train[[y]].asfactor()
valid[[y]] = valid[[y]].asfactor()
test[[y]] = test[[y]].asfactor()

File: code/load.mnist_enhanced.R
library(h2o)
h2o.init(nthreads=-1, max_mem_size="3G")

train = h2o.importFile("../datasets/mnist.enhanced_train.csv.gz")
valid = h2o.importFile("../datasets/mnist.enhanced_valid.csv.gz")
test = h2o.importFile("../datasets/mnist.enhanced_test.csv.gz")

x = 1:897
y = 898

train[,y] = as.factor(train[,y])
valid[,y] = as.factor(valid[,y])
test[,y] = as.factor(test[,y])

train=h2o.assign(train,"train")
valid=h2o.assign(valid,"valid")
test=h2o.assign(test,"test")
